Package.swift

```diff
@@ -6,8 +6,9 @@ import PackageDescription
 let package = Package(
     name: "Workflow",
     platforms: [
-        .iOS("14.0"),
+        .iOS("15.0"),
```

Comment:
We should be using symbolic references here:

```swift
        .iOS(.v15),
        .macCatalyst(.v15),
        .tvOS(.v15),
        .watchOS(.v8),
        .macOS(.v12),
```

Note: added macOS and catalyst references

Reply:
Sounds good: de1f0803